core/src/main/java/org/elasticsearch/cluster/routing/allocation/IndexMetaDataUpdater.java

```diff
@@ -95,7 +95,6 @@ public void relocationCompleted(ShardRouting removedRelocationSource) {
 
     @Override
     public void startedPrimaryReinitialized(ShardRouting startedPrimaryShard, ShardRouting initializedShard) {
```

Comment:
can you remove this method override here? No need to override empty method with empty method.